nemo/collections/asr/__init__.py

```diff
@@ -13,3 +13,14 @@
 # limitations under the License.
 
 from nemo.collections.asr import data, losses, models, modules
+
+from nemo.package_info import __version__
+
+# Set collection version equal to NeMo version.
+__version = __version__
+
+# Authorship.
+__author__ = "NVIDIA Corporation"
```

Comment:
Authorship makes very little sense here, i suggest simply removing the field entirely

Reply:
discussed this above, let's see what DL4RT will say.

Reply:
I think I missed that discussion, can you please point us to it?

Reply:
Chris explicitly wrote in email from Friday, June 26, 2020 4:40 PM:
"Script 1: If NeMo has collection metadata (human-readable descriptions, URLs, version strings…) we should retrieve that as well.
Proposed format is an array of dictionaries: [ { “id” : “Collection1” }, { “id” : “Collection2” }, … ]
If more information is available, add additional fields to the dictionary: { “id” : “Collection1”, “author” : “NVIDIA Corporation”, … }"

The discussion "what kind of metadata per collection" we want to send to nemo_editor is open, added _some fields_ to collection's `__init__.py` really as a proof of concept...